frontend/packages/local-storage-operator-plugin/OWNERS

```diff
@@ -0,0 +1,15 @@
+reviewers:
+  - a2batic
+  - afreen23
+  - bipuladh
+  - cloudbehl
+  - gnehapk
+  - rawagner
+approvers:
+  - afreen23
+  - bipuladh
+  - cloudbehl
+  - gnehapk
+  - rawagner
+labels:
+  - component/lso
```

Comment:
You need to send a separate PR for adding this label. Add the link to the description to this PR.

Reply:
Why is it so?

Reply:
Putting this here is fine. But I am assuming you want the label support in GitHub PRs too right? For that, you need to raise a separate PR.

Reply:
Got it. Thanks!